quilt/tools/store.py

```diff
@@ -396,13 +396,12 @@ def save_df(self, df, name, path, ext, target):
         """
         self._find_path_write()
         buildfile = name.lstrip('/').replace('/', '.')
-        storepath = os.path.join(self._pkg_dir, buildfile)
+        storepath = self._object_path('.' + buildfile)
```

Comment:
No, it just moves it from `quilt_packages/` to `quilt_packages/objs/` - that's what `self._object_path` returns.

I added a leading "." to make sure it won't conflict with hashes.